### PR TITLE
Cleaning up

### PR DESCRIPTION
**Pull request by ericgaspar**
Cleaning up variables

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by benneti** [comment]
I think something went wrong in this cleanup wrt. the database

File: scripts/install [hunk 1]
```diff
@@ -264,7 +230,7 @@ sleep 30
 
 # (Note that, by default, non-admins might not have your homeserver's permission to create communities.)
 if [ "$bot_synapse_adm" = true ]; then
-  ynh_psql_execute_as_root --database=$synapse_db_name --sql="UPDATE users SET admin = 1 WHERE name = ""$botname"";"
+  ynh_psql_execute_as_root --database=$db_name --sql="UPDATE users SET admin = 1 WHERE name = ""$botname"";"
```
Comment (Gredin67) [inline scripts/install:233]: indeed here it is not the mautrix db but the synapse db

Comment (Gredin67) [reply scripts/install:233]: -> revert change

**Review by Gredin67 — COMMENTED** [review]
revert line 233 (old 267)

**Comment by Gredin67** [comment]
!testme

**Comment by benneti** [comment]
is this ready for another test?

**Comment by benneti** [comment]
!testme

**Pull request closed without merging**